modules/nodes-pods-autoscaling-custom-pausing.adoc

```diff
@@ -39,13 +39,13 @@ apiVersion: keda.sh/v1alpha1
 kind: ScaledObject
 metadata:
   annotations:
-    autoscaling.keda.sh/paused-replicas: "4" <1>
-  creationTimestamp: "2023-02-08T14:41:01Z"
-  generation: 1
+    autoscaling.keda.sh/paused-replicas: '4' <1>
+  creationTimestamp: '2023-02-08T14:41:01Z'
+  generation: '1'
```

Comment:
please revert this, the quotoes are only needed for values in `spec.triggers.metadata`